### DISCUSSION TITLE
Why do I get an error when reading the search_index.json file?

### DISCUSSION THREAD

**Discussion by RemyYYZ** [squidfunk/mkdocs-material#7655] [Q&A] [answered]
### Error 
It has been confirmed that there is "1.21" text content in search_index.json, but no search results are displayed

But when I try to delete some text in search_index.json, it can be read normally

**So is this because the search_index.json file is too large?**

**Comment by RemyYYZ** [comment, marked as answer]
Solved

**Reply by kamilkrzyskow** [reply]
How was it solved @RemyYYZ?

**Reply by RemyYYZ** [reply]
> How was it solved @RemyYYZ?

I don't know. One day it recovered itself.😂
